models/iaf_psc_alpha.h

```diff
@@ -150,6 +183,7 @@ See also
 iaf_psc_delta, iaf_psc_exp, iaf_cond_exp
 
 EndUserDocs */
+// clang-format off
```

Comment:
```suggestion
EndUserDocs */
// clang-format on
```
Clang-format should probably be re-enabled again ;)

Reply:
Indeed, done :).